binding.gyp

```diff
@@ -6,7 +6,24 @@
   'targets': [
     {
       'target_name': 'iconv',
-      'include_dirs': ['<!(node -e "require(\'nan\')")'],
+      'cflags!': [ '-fno-exceptions' ],
+      'cflags_cc!': [ '-fno-exceptions' ],
+      'xcode_settings': { 
+        'GCC_ENABLE_CPP_EXCEPTIONS': 'YES',
+        'CLANG_CXX_LIBRARY': 'libc++',
```

Comment:
Can you remove this? It's incompatible with Node.js v6.x. node-iconv can just inherit it from node's common.gypi.